Hardcore.lua

```diff
@@ -219,6 +226,7 @@ function Hardcore:PLAYER_LOGIN()
 	_, class, _ = UnitClass("player")
 	PLAYER_NAME, _ = UnitName("player")
 	PLAYER_GUID = UnitGUID("player")
+  PLAYER_FACTION, _ = UnitFactionGroup("player")
```

Comment:
will reformat -- this is using 2 spaces

Reply:
I've been told we should be using tab = 4spc, and we'll worry about linting later